package.json

```diff
@@ -216,11 +216,11 @@
     "websocket": "1.0.34",
     "wireit": "0.14.4",
     "ws": "8.16.0",
-    "yargs": "17.7.2",
-    "zod": "3.22.4"
+    "yargs": "17.7.2"
   },
   "dependencies": {
     "mitt": "3.0.1",
-    "urlpattern-polyfill": "10.0.0"
+    "urlpattern-polyfill": "10.0.0",
```

Comment:
hmm, now I remember. These modules are only bundled, therefore, they were devDependencies since you do not actually need them once you install from npm.

Reply:
back to devDependencies?

Reply:
I am not sure as in theory you can run the mapper instance and send commands that way, and that has Zod as production dependency. That's all bundled in the output folder.